source/backups.rst

```diff
@@ -148,7 +148,7 @@ Following steps are needed to restore a previously saved backup:
 
    ::
 
-      ./deploy/backup/restore-backup.sh <backup-name> <cluster-name>
+      kubectl apply -f deploy/backup/restore.yaml 
```

Comment:
Could you also add an example like in lines 111-124

Reply:
Updated.

Reply:
sorry, still don't see example 😞